ambry-store/src/test/java/com.github.ambry.store/StoreTestUtils.java

```diff
@@ -109,6 +110,11 @@ public boolean isSealed() {
       return isSealed;
     }
 
+    @Override
+    public JSONObject getSnapshot() {
+      return new JSONObject();
```

Comment:
just curious why here returns new `JSONObject`, however, in previous test, `null` is directly returned?

Reply:
No preference. Made this return `null` too.